Services/EmailSender.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.AspNetCore.Identity.UI.Services;
+using Microsoft.Extensions.Options;
+using System.Net;
+using System.Net.Mail;
+using System.Threading.Tasks;
+
+namespace Personaltool.Services
+{
+    public class EmailSender : IEmailSender
+    {
+        public EmailSender(IOptions<AuthMessageSenderOptions> optionsAccessor)
+        {
+            Options = optionsAccessor.Value;
+        }
+
+        public AuthMessageSenderOptions Options { get; } //set only via Secret Manager
+
+        public Task SendEmailAsync(string sentTo, string subject, string message)
+        {
+            // Hier den E-Mail-Dienst einfügen, um eine E-Mail-Nachricht zu senden.
```

Comment:
We should keep all comments either german or english, I'd prefer english

Reply:
comment now in english